spec/features/profiles_spec.rb

```diff
@@ -74,3 +74,36 @@
     end
   end
 end
+
+
+feature 'Websites New' do
+  describe "Form" do
+    before :each do
+      Website.create(
+        name: 'Sample Website',
+        url: 'http://www.sample.com/'
+      )
+      visit '/profiles/new'
+    end
+
+    it "has forms" do
```

Comment:
Test what you see, not the structure of the page.

http://betterspecs.org/#integration

Avoid using capybara selectors to make your tests less brittle.  What if you change the ID of the input profile_name to something else later?  This test would break, even if nothing else about the behavior or intent of the form has changed.  

healthy debate about feature specs, aka integration specs.  https://github.com/lelylan/betterspecs/issues/14

Also, great article on avoiding capybara selectors all together... https://blog.codeship.com/principles-of-effective-testing-with-capybara/